console/package.json

```diff
@@ -18,8 +18,8 @@
   },
   "main": "src/main.js",
   "scripts": {
-    "start": "electron . --local-debug-builds",
-    "local-release": "electron . --local-release-builds",
+    "start": "electron . --local-debug-builds --debug",
+    "local-release": "electron . --local-release-builds --debug",
```

Comment:
we may want to remove --debug from here eventually, but I've been running Release ds+acs so this is useful for me.